aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -195,6 +197,12 @@ module aptos_framework::storage_gas {
         per_byte_write: u64,
     }
 
+    /// The quota in bytes that is free of storage gas charge.
+    struct FreeGasQuota has key {
+        bytes_read: u64,
+        bytes_write: u64,
+    }
+
```

Comment:
Oh no... I forgot that we need to introduce a new struct for this...

Reply:
What else can I do... I cannot add new field...

Reply:
Yeah this is probably a bit trickier than I initially thought it would be. Another related issue is how we should version/feature-gate this. The VM needs to know if it needs to load this on-chain config

Reply:
The new struct should be alright IMO? This will be a lot easier once we have multistep proposal. We can then do a single proposal with 2 steps: (1) to upgrade the storage_gas module and (2) to call set_free_quota, which instantiates this new struct.

The logic inside the VM can be gated by a feature flag. This can even be chained with the previously mentioned multistep proposal to enable the feature.